lookout/style/format/features.py

```diff
@@ -269,7 +269,7 @@ def __call__(self, sibling: Union[VirtualNode, bblfsh.Node], node: VirtualNode
         if bblfsh_node:
             for role_id in bblfsh_node.roles:
                 role = bblfsh.role_name(role_id)
-                role_indices[self._roles_index[role]] += 1
+                role_indices[self._roles_index[role]] = 1
```

Comment:
Sorry I'm confused then. @zurk opened an issue because we had code to handle roles that appear several times (https://github.com/src-d/style-analyzer/issues/221) and I indeed thought it happened but then when I checked the log of when we discussed the issue (https://src-d.slack.com/archives/C7USX021L/p1537862678000100?thread_ts=1537861320.000100&cid=C7USX021L) I couldn't see any duplicates. I also checked the doc just now and couldn't find any mention of duplicates (https://doc.bblf.sh/uast/uast-specification.html#roles). I'll put the + again if indeed some roles are used several time by design (not just because of a bug) :)

Reply:
After asking on #devLA it's not a feature so it's safe to use only +1 and to consider roles as a set.

Reply:
https://github.com/bblfsh/python-driver/issues/130#issuecomment-341648217:
>it's legit that it also get the FUNCTION,CALL applied (since it's a function call) so we would end with 2 FUNCTION roles anyway.

There was a bug when wrong roles are duplicated but AFAIK some roles are duplicated by design.

Reply:
>Juanjo Álvarez [9:47 AM]
roles shouldn't be duplicated, there is actually a pass on the annotations to deduplicate them

not anymore - it's safe to use just assigment right now